src/wallet/walletdb.cpp

```diff
@@ -832,6 +845,16 @@ DBErrors WalletBatch::LoadWallet(CWallet* pwallet)
                     // Set tx_corrupt back to false so that the error is only printed once (per corrupt tx)
                     wss.tx_corrupt = false;
                     result = DBErrors::CORRUPT;
+                } else if (wss.descriptor_unknown) {
+                    strErr = "Error: Unrecognized descriptor found in wallet %s\n";
```

Comment:
Missing `strprintf` for the `%s` here

Reply:
thx pushed.

> Also I think we should avoid having \ns here, makes the log look weird. If you want to have multiline, then each of these needs to be WalletLogPrintf'd separately.

pushed as well, it was a remanent of the error propagation work.

Note: same as happen with the other db load errors, we log the messages twice in some flows (e.g. loading an unrecognized descriptor at startup). The text that is formatted with multiline for the GUI (the one inside `CWallet::Create`) will be logged right after the `WalletBatch::LoadWallet` logging. This is due the `ThreadSafeMessageBox` slot that logs the string if it's marked "insecure" as well.
Will keep the plan and tackle this in a follow-up PR.